service/kvmconfig/framework.go

```diff
@@ -89,118 +84,55 @@ func NewFramework(config FrameworkConfig) (*framework.Framework, error) {
 		}
 	}
 
-	v2ResourceSetHandles := func(obj interface{}) bool {
-		customObject, err := v2key.ToCustomObject(obj)
-		if err != nil {
-			return false
-		}
-		versionBundleVersion := v2key.VersionBundleVersion(customObject)
+	var newInformer *informer.Informer
+	{
+		c := informer.DefaultConfig()
 
-		if versionBundleVersion == "1.0.0" {
-			return true
-		}
-		if versionBundleVersion == "0.1.0" {
-			return true
-		}
-		if versionBundleVersion == "" {
-			return true
-		}
+		c.Watcher = config.G8sClient.ProviderV1alpha1().KVMConfigs("")
 
-		return false
+		newInformer, err = informer.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
 	}
 
-	var v2Resources []framework.Resource
+	var v2ResourceSet *framework.ResourceSet
 	{
-		c := v2.ResourcesConfig{
+		c := v2.ResourceSetConfig{
 			CertsSearcher:      certsSearcher,
 			K8sClient:          config.K8sClient,
 			Logger:             config.Logger,
 			RandomkeysSearcher: randomkeysSearcher,
 
+			HandledVersionBundles: []string{
+				"1.1.0",
+			},
```

Comment:
I think it would be better to keep the version bundle versions close to the resources. This was confusing me already. Usually you want to know which resources are executed on which version bundle. Having this in the same place makes the understanding easier and the interface for the resource set creation is cleaner. In the versioned folder we can have a common variable for the list and later a common constant because there should actually only be one version per resource list.

Reply:
I think other way around. I'd like to go the framework.go and see immediately which bundles are handled by which resources. In a _single_ file. No jumping around.

Reply:
I see, because the resource set is versioned after the resources version. Then lets go with this.